entrypoint.sh

```diff
@@ -83,18 +83,31 @@ fi
 # We assume the the patch/Z version is at least 11, based on the current edit
 # Thus it will definitely have both the RBAC and create_user features
 AIRFLOW_VERSION="$(airflow version)"
+AIRFLOW_X_VERSION="$(echo ${AIRFLOW_VERSION} | cut -d . -f 1)"
 AIRFLOW_Y_VERSION="$(echo ${AIRFLOW_VERSION} | cut -d . -f 2)"
 
+echo "${AIRFLOW_VERSION}, ${AIRFLOW_X_VERSION}, ${AIRFLOW_Y_VERSION}"
```

Comment:
Think can remove this line? Seems to be a debug line haha.

Reply:
Oops i overlooked this